firestore/cloud-client/snippets_test.py

```diff
@@ -235,6 +235,10 @@ def test_order_where_limit():
     snippets.order_where_limit()
 
 
+def test_order_limit_to_last():
+    snippets.order_limit_to_last()
```

Comment:
Does it make sense to check the output?

Reply:
@IlyaFaer Could you comment on this?

You can return the results in the function and check them here if that's easier than checking the string output.

```py
results = snippets.order_limit_to_last()
assert ...
```

Reply:
Oh, that slipped my radars. Yeah, I thought about that, but most of the snippets test are written this way - just a call to snippet function. So, as I see it, if we're gonna add an `assert`, we should do this for all the tests to keep the fashion, which probably should be done in a separate issue. Adding an `assert` into this single test, while other similar tests doesn't have it, looks weird. WDYT?

Reply:
Alright.

Reply:
Can you create an issue for it?

Reply:
Sure, will make a /cc to you when created. And I think I'll do this